src/components/Tabs/Tabs.tsx

```diff
@@ -13,8 +13,6 @@ import React, { forwardRef, useState } from "react";
 
 import Button from "../Button/Button";
 import Icon from "../Icons/Icon";
-import useCarouselStyles from "../../hooks/useCarouselStyles";
```

Comment:
Can you update `src/hooks/useCarouselStyles.mdx` and remove the references to the `Tabs` component and the code snippet?

This was the only component using this so I think we can do without this hook, but that can be an update for the future.